docs/series/LAMP-server.md

```diff
@@ -0,0 +1,241 @@
+# Deploy a LAMP server
+
+The **LAMP** stack, comprising **L**inux, **A**pache, **M**ySQL, and **P**HP owes much of its popularity and versatility to its open-source nature.
+
+- **L**inux, the operating system at its core, embodies collaborative development, providing a stable foundation for web servers.
+
+- **A**pache, the iconic web server software, benefits from a community-driven approach, ensuring flexibility and reliability.
+
+- **M**ySQL, the relational database management system, democratizes access to robust database technology, thanks to its open-source ethos.
+
+- **P**HP, thrives on community contributions, facilitating dynamic and interactive web experiences.
+
+Together, the LAMP stack exemplifies the power of open-source collaboration, empowering developers worldwide to create scalable, cost-effective web applications that drive the modern internet.
+
+## Introduction
+
+For the sake of this guide we'll focus on a **LAMP** server that is composed of these four components:
+
+| Name | Description |
+|----------|----------|
+| (**L**) AlmaLinux    | The operating system, in this case AlmaLinux 8   |
+| **A**pache 2   | A trusted open-source web server software known for its reliability and scalability.  |
+| **M**ariaDB  | A popular open-source database system for managing structured data efficiently.   |
+| **p**hp-fpm   | A widely-used web scripting language known for its simplicity, flexibility, and strong community.   |
```

Comment:
This is very pedantic, but `php-fpm` isn't `PHP`, it's the FastCGI process manager.

Reply:
Well, yeah you're right, I'll tweak this to be more accurate.